ROMFS/px4fmu_common/init.d-posix/airframes/CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ px4_add_romfs_files(
 
 	4001_gz_x500
 	4002_gz_x500_depth
-	4003_gz_plane
+	4003_gz_rascal_110
```

Comment:
Could you clarify why this is the rascal 110? The mesh used for the model doesnt seem to be the case.

Reply:
![image](https://user-images.githubusercontent.com/10233412/214372581-f44a165d-041d-4273-9981-90ba7e63fbfb.png)
It was originally created off of it ages ago with the values representing it.

Reply:
When I added the control surfaces to the model (in https://github.com/PX4/PX4-SITL_gazebo-classic/pull/476), this came from a cessna model in https://github.com/osrf/gazebo_models/tree/master/cessna, so I was under the impression that the mesh was a scaled down cessna. 

However, could we keep the plane naming? I don't see why we want to call this model the rascal - I am pretty sure the mesh is not the rascal 110. The rascal 110 has an elliptical wing shape(https://sigmfg.com/products/sig-rascal-110egarf), and the control surfaces also look different. Also, the aerodynamic parameters are non-sensical, so it would definitely not fly like the real model.

Reply:
Back in 2016 when the model was added there wasn't a perfect "exact cad model" of the rascal 100, so the model was stretched and modified to have the same relative properties such as 1.1m wingspan you can currently see it has. (Hence the 110 in the name and the size of the size of the model aircraft). This is meant to represent that aircraft, even if the visuals on the wings are slightly different in terms of tip geometry.

Reply:
I still don't see the benefits of naming the model as the `rascal` unless we model the dynamics of the real plane into the simulation. If it does not represent the real model, I would be against trying to anchor on a specific model which is misleading.

Reply:
These were the values it was previously given: 
```xml
      <inertial>
        <pose>0 0 0 0 0 0</pose>
        <mass>1.5</mass>
        <inertia>
          <ixx>0.197563</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>0.1458929</iyy>
          <iyz>0</iyz>
          <izz>0.1477</izz>
        </inertia>
      </inertial>
```
and

```xml
<plugin name="left_wing" filename="libLiftDragPlugin.so">
      <a0>0.05984281113</a0>
      <cla>4.752798721</cla>
      <cda>0.6417112299</cda>
      <cma>-1.8</cma>
      <alpha_stall>0.3391428111</alpha_stall>
      <cla_stall>-3.85</cla_stall>
      <cda_stall>-0.9233984055</cda_stall>
      <cma_stall>0</cma_stall>
      <cp>-0.05 0.3 0.05</cp>
      <area>0.12</area>
      <air_density>1.2041</air_density>
      <forward>1 0 0</forward>
      <upward>0 0 1</upward>
      <link_name>base_link</link_name>
      <control_joint_name>
        left_elevon_joint
      </control_joint_name>
      <control_joint_rad_to_cl>-0.5</control_joint_rad_to_cl>
    </plugin>
    <plugin name="right_wing" filename="libLiftDragPlugin.so">
      <a0>0.05984281113</a0>
      <cla>4.752798721</cla>
      <cda>0.6417112299</cda>
      <cma>-1.8</cma>
      <alpha_stall>0.3391428111</alpha_stall>
      <cla_stall>-3.85</cla_stall>
      <cda_stall>-0.9233984055</cda_stall>
      <cma_stall>0</cma_stall>
      <cp>-0.05 -0.3 0.05</cp>
      <area>0.12</area>
      <air_density>1.2041</air_density>
      <forward>1 0 0</forward>
      <upward>0 0 1</upward>
      <link_name>base_link</link_name>
      <control_joint_name>
        right_elevon_joint
      </control_joint_name>
      <control_joint_rad_to_cl>-0.5</control_joint_rad_to_cl>
    </plugin>
    <plugin name="elevator" filename="libLiftDragPlugin.so">
      <a0>-0.2</a0>
      <cla>4.752798721</cla>
      <cda>0.6417112299</cda>
      <cma>-1.8</cma>
      <alpha_stall>0.3391428111</alpha_stall>
      <cla_stall>-3.85</cla_stall>
      <cda_stall>-0.9233984055</cda_stall>
      <cma_stall>0</cma_stall>
      <cp>-0.5 0 0</cp>
      <area>0.01</area>
      <air_density>1.2041</air_density>
      <forward>1 0 0</forward>
      <upward>0 0 1</upward>
      <link_name>base_link</link_name>
      <control_joint_name>
        elevator_joint
      </control_joint_name>
      <control_joint_rad_to_cl>-4.0</control_joint_rad_to_cl>
    </plugin>
    <plugin name="rudder" filename="libLiftDragPlugin.so">
      <a0>0.0</a0>
      <cla>4.752798721</cla>
      <cda>0.6417112299</cda>
      <cma>-1.8</cma>
      <alpha_stall>0.3391428111</alpha_stall>
      <cla_stall>-3.85</cla_stall>
      <cda_stall>-0.9233984055</cda_stall>
      <cma_stall>0</cma_stall>
      <cp>-0.5 0 0.05</cp>
      <area>0.02</area>
      <air_density>1.2041</air_density>
      <forward>1 0 0</forward>
      <upward>0 1 0</upward>
      <link_name>base_link</link_name>
    </plugin>
```

Reply:
@bperseghetti Do you know where those parameters come from? Could you also point to where the model comes from? all the references I found regarding the rascal 110 has a wingspan of 2.74m